docs/book/src/development/releasing.md

```diff
@@ -4,25 +4,26 @@
 
 1. Make sure your repo is clean by git's standards
 2. If this is a new minor release, create a new release branch and push to github, otherwise switch to it, for example `release-0.2`
-3. Run `make release-notes` to gather changes since the last revision. If you need to specify a specific tag to look for changes
-   since, use `make release-notes ARGS="--from <tag>"` Pay close attention to the `## :question: Sort these by hand` section, as it contains items that need to be manually sorted.
-4. Tag the repository and push the tag `git tag -s -m $VERSION $VERSION`
-5. Create a draft release in github and associate it with the tag that was just created, copying the generated release notes into
-   the draft.
-6. Checkout the tag you've just created and make sure git is in a clean state
-7. Run `make release`
-8. Attach the files to the drafted release:
+3. Tag the repository and push the tag `git tag -s -m $VERSION $VERSION`
+4. Set an environment variable PREVIOUS_VERSION to the last release tag.
+5. Create a draft release in github and associate it with the tag that was just created, copying the generated release notes
```

Comment:
What is the difference of this with `make create-gh-release`?

Reply:
None, cleaned up.